catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ScoreCommand.java

```diff
@@ -0,0 +1,253 @@
+    protected PrintStream console = System.out;
+
+    private static final String ERROR_COLOR = COLOR_RED;
+
+    // Put all the formatting in one place so it's consistent and easy to change
+    private static final int COLUMN_1_WIDTH = 20;
+    private static final int COLUMN_2_WIDTH = 47;
+    private static final int COLUMN_3_WIDTH = 5;
+    private static final String TITLE = StringUtils.center("Metadata Quality Report", 80);
+    private static final String SEPARATOR = StringUtils.repeat("-", COLUMN_1_WIDTH) + "-+-" + StringUtils.repeat("-", COLUMN_2_WIDTH) + "-+-" + StringUtils.repeat("-", COLUMN_3_WIDTH);
+    private static final String TOTAL = StringUtils.repeat(" ", COLUMN_1_WIDTH + COLUMN_2_WIDTH + COLUMN_3_WIDTH - 7) + "Total  ";
+    private static final String FORMAT = "%-" + COLUMN_1_WIDTH + "s | %-" + COLUMN_2_WIDTH + "s | %" + COLUMN_3_WIDTH + "s %n";
+
+
+    @Override
+    public Object execute() throws Exception {
+
+        try {
+            File file = new File(ruleSetFilename);
+            InputStream ruleSetInputStream = new FileInputStream(file);
+            ruleSet = new ValidatorConfigReader(ruleSetInputStream).getRuleSet();
+        } catch (Exception e) {
+            console.printf(ERROR_COLOR + "Ruleset not configured.%n" + COLOR_DEFAULT, filename);
+            LOGGER.error("Error trying to configure ruleset {}", ruleSetFilename, e);
+            return null;
+        }
+
+        if (fileExists() && hasValidators()) {
+            Metacard metacard = createMetacard();
+            if (metacard != null) {
+                score(metacard);
+            }
+        }
+
+        // We must return an object per the Action contract which
+        // this class implements. It is sufficient to return null.
+        return null;
+    }
+
+    private boolean fileExists() {
+        File fileToValidate = new File(filename);
+        if (!fileToValidate.exists()) {
+            console.printf(ERROR_COLOR +
+                    "Unable to locate file '%s'. Double check the path is correct and the file exists.%n"
+                    + COLOR_DEFAULT, filename);
+            return false;
+        }
+
+        if (!fileToValidate.isFile()) {
+            console.printf(ERROR_COLOR + "'%s' is not a file.%n" + COLOR_DEFAULT, filename);
+            return false;
+        }
+        return true;
+    }
+
+    private boolean hasValidators() {
+        if (validators == null || validators.size() < 1) {
+            console.println(ERROR_COLOR + "No validators have been configured" + COLOR_DEFAULT);
+            return false;
+        }
+        return true;
+    }
+
+    private Metacard createMetacard() {
+
+        // Although we are given XML, we need to put it in a metacard to work with the validators.
+        MetacardImpl metacard = null;
+        try {
+            String metadata = IOUtils.toString(new File(filename).toURI());
+            metacard = new MetacardImpl();
+            metacard.setMetadata(metadata);
```

Comment:
Seems odd to not call an input transformer for this document.  It seems we would want to score a metacard not just it's metadata as different MetacardTransformers will produce different outputs (could be JSON, various XML formats, ATOM, KML, etc) as a whole rather than just the metadata.  This will only validate something that can be ingested but not necessarily the output to a client.